spec/apis/path-to-active-when.spec.js

```diff
@@ -2,182 +2,121 @@ import { pathToActiveWhen } from "single-spa";
-        toDynamicPathValidatorRegex("/#/pathname").test(
-          "/#/pathname/anything/everything"
-        )
-      ).toBe(true);
-    });
-
-    it("Should generate correct regex for '/#/pathname/:dynamic/notDynamic'", () => {
-      expect(
-        toDynamicPathValidatorRegex("/#/pathname/:dynamic/notDynamic").test(
-          "/#/pathname/1/notDynamic"
-        )
-      ).toBe(true);
-      expect(
-        toDynamicPathValidatorRegex("/#/pathname/:dynamic/notDynamic").test(
-          "/#/pathname/1/notDynamicExtra"
-        )
-      ).toBe(false);
-      expect(
-        toDynamicPathValidatorRegex("/#/pathname/:dynamic/notDynamic").test(
-          "/#/pathname/1/notDynamic/"
-        )
-      ).toBe(true);
-      expect(
-        toDynamicPathValidatorRegex("/#/pathname/:dynamic/notDynamic").test(
-          "/#/pathname//notDynamic/anything/everything"
-        )
-      ).toBe(false);
-    });
-
-    it("Should generate correct regex for '/pathname/:dynamic/notDynamic'", () => {
-      expect(
-        toDynamicPathValidatorRegex("/pathname/:dynamic/notDynamic").test(
-          "/pathname/1/notDynamic"
-        )
-      ).toBe(true);
-      expect(
-        toDynamicPathValidatorRegex("/pathname/:dynamic/notDynamic").test(
-          "/pathname/1/notDynamic/anything/everything"
-        )
-      ).toBe(true);
-      expect(
-        toDynamicPathValidatorRegex("/pathname/:dynamic/notDynamic").test(
-          "/pathname//notDynamic"
-        )
-      ).toBe(false);
-    });
-
-    it("Should generate correct regex for ''", () => {
-      expect(toDynamicPathValidatorRegex("").test("")).toBe(true);
-      expect(toDynamicPathValidatorRegex("").test("/")).toBe(true);
-      expect(toDynamicPathValidatorRegex("").test("/anything/everything")).toBe(
-        true
-      );
-    });
-
-    it("Should generate correct regex for '/'", () => {
-      expect(toDynamicPathValidatorRegex("").test("")).toBe(true);
-      expect(toDynamicPathValidatorRegex("").test("/")).toBe(true);
-      expect(toDynamicPathValidatorRegex("").test("/anything/everything")).toBe(
-        true
-      );
-    });
-
-    it("Should generate correct regex for ':dynamic/:dynamic'", () => {
-      expect(toDynamicPathValidatorRegex(":dynamic/:dynamic").test("1/1")).toBe(
-        true
-      );
-      expect(
-        toDynamicPathValidatorRegex(":dynamic/:dynamic").test("1/1/")
-      ).toBe(true);
-      expect(
-        toDynamicPathValidatorRegex(
-          ":dynamic/:dynamic/anything/everything"
-        ).test("1/1/anything/everything")
-      ).toBe(true);
-      expect(toDynamicPathValidatorRegex(":dynamic/:dynamic").test("1//")).toBe(
-        false
-      );
-      expect(toDynamicPathValidatorRegex(":dynamic/:dynamic").test("1/")).toBe(
-        false
-      );
-      expect(toDynamicPathValidatorRegex(":dynamic/:dynamic").test("1")).toBe(
-        false
-      );
-    });
-
-    it("Should generate correct regex for '$🎉/:dynamic$🎉'", () => {
```

Comment:
Was this test intentionally deleted?

Reply:
Yes, this is debatable, although it's possible I thought having only real world scenarios here would make the tests better... I could return them, no problem 👍

Reply:
My understanding is the web URLs only support ascii characters and don't support emojis, so I think this is fine to omit.